### PR TITLE
feat: Add Osmosis cross chain swap outpost example and README

### PR DESCRIPTION
**Pull request by Vvaradinov**

### CODE REVIEW

File: outposts/osmosis/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Osmosis Cross Chain Swap (XCS) Outpost
+
+## Overview
```
Comment (danburck) [inline outposts/osmosis/README.md:3]: @Vvaradinov please apply the same markdown linter as on evmos/Evmos

File: outposts/osmosis/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+The following is an example Osmosis Outpost contract that interacts with the 
+[Osmosis Cross Chain Swap contract](https://github.com/osmosis-labs/osmosis/tree/be63fb58580b87808f6d7ed9523a43e976899258/cosmwasm/contracts/crosschain-swaps),
+and specifically version 1 of the contract for the [**Native to Osmosis Native swap**](https://github.com/osmosis-labs/osmosis/tree/main/cosmwasm/contracts/crosschain-swaps#non-native-to-non-native). 
+In this case you can swap Evmos for Osmosis tokens and receive them back on the Evmos chain.
```
Comment (danburck) [inline outposts/osmosis/README.md:7]: Add that the outpost is accessible from smart contracts.

File: outposts/osmosis/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+This contract is configured for testnet and would **NOT** work on mainnet without changing the configurations.
+The channel id, contract address and pool ids are all testnet specific. 
+For mainnet this document and the contract will be updated with the correct values.
+
+**NOTE** - Moreover the [evmos/osmo pool 48](https://testnet.osmosis.zone/pool/48) was setup in a way that 1 aevmos = 1 osmo.
+On mainnet this will not be the case as it will take into consideration the different token precisions and the swap will be adjusted.
+On mainnet: evmos token precision = 18, osmo token precision = 6.
+
+
+## Mainnet
```
Comment (danburck) [inline outposts/osmosis/README.md:35]: Add disclaimer, that this context isn't audited or production ready

**Review by danburck — APPROVED** [review]
LGTM @Vvaradinov 

Left bit comments

**Review by MalteHerrmann — APPROVED** [review]
Looks great! 🚀 I would suggest to add the Solidity style comments as I wasn't sure what to exactly put into the function arguments

**Pull request merged**